src/status_im2/contexts/syncing/events.cljs

```diff
@@ -36,6 +37,24 @@
                                        :custom-bootnodes-enabled? false}}]
     (node/get-multiaccount-node-config db)))
 
+(defn- includes-error?
+  [json-str]
+  (let [data (-> json-str
+                 js/JSON.parse
+                 (js->clj :keywordize-keys true))]
+    (-> (get data :error)
+        (#(and (not (nil? %)) (boolean (seq %)))))))
```

Comment:
We can use the `json->clj` from `utils.transform` ns as it has the `keywordize-keys` config.

Additionally, we can write this function

```clojure
(defn- includes-error?
  [json-str]
  (-> json-str
      transforms/json->clj
      (get :error "") ;; we set the empty string as a fallback so that we can skip the `nil?` check
      not-empty)) ;; `not-empty` on empty string would result in `nil`

```